### PR TITLE
[TTAHUB-2163] Add session objective component to the RTR

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
The RTR can show session objectives. There isn't a way to connect them to a goal yet; that depends on work done in #1911. Since I'm not sure yet _exactly_ what work will be needed to fill in the gaps, this PR doesn't try to. It just creates the render function for the session objective.

## How to test
Uncomment lines 717-730 in /src/services/recipient.js. This will return a static session objective to the RTR goal query. The first goal for every recipient should have a session objective in there for you to take a look at.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2163

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -15,6 +15,13 @@ a:visited {
   color: $text-visited
 }
 
+/*|||||| Accessibility fix |||||||*/
+[hidden] {
+  display: none !important;
```
Comment (thewatermethod) [inline frontend/src/App.scss:20]: I don't know why this wasn't already in there

File: frontend/src/components/GoalCards/ObjectiveCard.scss [hunk 2]
```diff
@@ -1,3 +0,0 @@
-.ttahub-goal-card__objective-list .usa-list.usa-list--unstyled {
```
Comment (thewatermethod) [inline frontend/src/components/GoalCards/ObjectiveCard.scss:1]: deleted; applied via helper class

File: frontend/src/components/GoalCards/SessionObjectiveCard.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    grantNumbers,
+    endDate,
+    topics,
+    sessionName,
+  } = objective;
+
+  const trainingReportUrl = `/training-report/${trainingReportId.substring(trainingReportId.lastIndexOf('-') + 1)}/event-summary`;
+
+  return (
+    <dl className="ttahub-goal-card__objective-list ttahub-goal-card__objective-list--session-objective usa-list usa-list--unstyled padding-2 margin-top-2 border radius-lg" hidden={!objectivesExpanded}>
```
Comment (thewatermethod) [inline frontend/src/components/GoalCards/SessionObjectiveCard.js:22]: I think a ```<dl>``` is probably the right tag to use on the other Objective card as well, but I'm not sure about some of the markup we have nested inside of it being appropriate, so I didn't refactor that one just yet.

File: src/services/recipient.js [hunk 4]
```diff
@@ -710,6 +710,25 @@ export async function getGoalsByActivityRecipient(
     },
   });
 
+  // TODO - remove: this is for testing purposes only,
+  // and to demonstrate the schema required
+  // to be removed when the backend is complete or before merging, whichever is first
+
```
Comment (thewatermethod) [inline src/services/recipient.js:716]: uncomment the below to view the session objective on the RTR

File: frontend/src/App.scss (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/App.scss:20]: Do we need !important

File: frontend/src/components/GoalCards/SessionObjectiveCard.js (hunk 3 above)
Comment (AdamAdHocTeam) [reply frontend/src/components/GoalCards/SessionObjectiveCard.js:22]: I guess I could go either way to seems its terms and their definitions for <dl>

The <dl> tag defines a description list. The <dl> tag is used in conjunction with <dt> (defines terms/names) and <dd> (describes each term/name).

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good. Not sure if this is only half the change, feel free to re request my review if you add more.

**Pull request merged**